root/etc/e-smith/templates/etc/nethserver/otp.force/10base

```diff
@@ -0,0 +1,17 @@
+{
+  use strict;
+  use warnings;
+  use esmith::ConfigDB;
+
+  my $db = esmith::ConfigDB->open_ro();
+  my $domain = $db->get_value('DomainName');
+
+  foreach my $name (glob('/var/lib/nethserver/2fa/*')) {
+      $name =~ s|/var/lib/nethserver/2fa/||g;
+      $OUT .= "$name\n";
+      # we need to display user and user@domain.org
+      next if ($name eq 'root');
+      $name =~ s/\@$domain$//;
```

Comment:
use global variable available in all templates:
```suggestion
      $name =~ s/\@$DomainName$//;
```

Reply:
done